dist/index.html

```diff
@@ -42,11 +45,62 @@
     <script type="text/javascript" src="itkVtkViewer.js">
     </script>
     <script>
-      var container = document.querySelector('.content');
-      var viewer = itkVtkViewer.processURLParameters(container, { fullscreen: true });
+     async function setupViewerForImJoy(container) {
+      const imjoyRPC = await imjoyLoader.loadImJoyRPC({
+        api_version: '0.2.3'
+      })
+      const api = await imjoyRPC.setupRPC({
+        name: 'itk-vtk-viewer',
+        version: '9.23.x',
```

Comment:
@oeway we will get `version` from *package.json*.

Reply:
Yes, that will be great!